frontend/src/components/AccountSendTable.vue

```diff
@@ -0,0 +1,184 @@
+  >
+    <!-- Card Layout for grid option -->
+    <template v-slot:item="props">
+      <div :key="props.row.dateSentUnix" class="col-12">
+        <q-card class="card-border cursor-pointer q-pa-md col justify-center items-center">
+          <q-card-section class="row justify-center items-center">
+            <img class="q-mr-md" :src="props.row.tokenLogo" style="width: 1.2rem" />
+            <div class="text-primary text-h6 header-black q-pb-none">
+              {{ props.row.amount }} {{ props.row.tokenSymbol }}
+            </div>
+          </q-card-section>
+          <div class="row justify-between items-center">
+            <div>
+              <span class="q-mr-xs">{{ $t('AccountSendTable.receiver') }}</span>
+            </div>
+            <div>
+              <div @click="copyAddress(props.row.address)" class="cursor-pointer copy-icon-parent">
+                <span>{{ props.row.addressShortened }}</span>
+                <q-icon color="primary" class="q-ml-sm" name="far fa-copy" />
+              </div>
+            </div>
+          </div>
+
+          <div class="row justify-between items-center text-caption text-grey">
+            <div>{{ $t('AccountSendTable.received') }}</div>
+            <div>
+              {{ props.row.dateSent }}
+              {{ props.row.dateSentTime }}
+            </div>
+          </div>
+        </q-card>
+      </div>
+    </template>
+
+    <!-- Body row configuration -->
+    <template v-slot:body="props">
+      <q-tr :props="props" :key="props.row.dateSentUnix">
+        <q-td v-for="col in props.cols" :key="col.name" :props="props">
+          <!-- Date column -->
+          <div v-if="col.name === 'dateSent'" class="d-inline-block">
+            <div
+              @click="openInEtherscan(props.row.hash)"
+              class="row justify-start items-center cursor-pointer external-link-icon-parent"
+            >
+              <div class="col-auto">
+                <div>{{ col.value }}</div>
+                <div class="text-caption text-grey">{{ props.row.dateSentTime }}</div>
+              </div>
+              <q-icon class="external-link-icon" name="fas fa-external-link-alt" right />
+            </div>
+          </div>
+
+          <!-- Amount column -->
+          <div v-else-if="col.name === 'amount'">
+            <div class="row justify-start items-center no-wrap">
+              <img class="col-auto q-mr-md" :src="props.row.tokenLogo" style="width: 1.2rem" />
+              <div class="col-auto">
+                {{ col.value }}
+                {{ props.row.tokenSymbol }}
+              </div>
+            </div>
+          </div>
+
+          <!-- Sender column -->
+          <div v-else-if="col.name === 'from'" class="d-inline-block">
+            <div @click="copyAddress(props.row.address)" class="cursor-pointer copy-icon-parent">
+              <span>{{ props.row.addressShortened }}</span>
+              <q-icon class="copy-icon" name="far fa-copy" right />
```

Comment:
If we have an ENS name, let's show that as the main content here instead, and we can show the shortened address underneath in smaller/grey text